changelog.md

```diff
@@ -1,5 +1,32 @@
 
 
+# 2.6.0 /2020-01-xx
+
+### New Feature
+- Multi-upload file directly available in tracim frontend
+- Page title show where you are in Tracim
+- Its now possible to delete user and anonymise information about him/her with tracimcli command
+- Preview generator now support raw file and 3D file (working with xvfb in docker container)
+- With proxy apache: you can now used browser cache policy. If you used Tracim with Docker you need to backup and delete existing apache configuration file in `~/tracim/etc/` (default path with Tracim Docker image). New file is created on next startup with new parameter inside.
+- All apps are now enabled/disabled directly with one parameter and its more simple to develop new app
+
+### Fixed issues
+- Gallery: #2540, #2541, #2551, #2574, #2583 
+- Frontend: #1396, #1560, #1656, #2607, #2608, #2611, #2641
+- Backend: #2588, #2570, #2610, #2129, #1484, #2010
+
+### Breaking/Important change
+- `agenda`, `share_content`, `upload_permission` and `collaborative_document_edition` are now real applications, you can add/remove them from `app.enabled` list and its add/remove the associated feature 
```

Comment:
```suggestion
- `agenda`, `share_content`, `upload_permission` and `collaborative_document_edition` are now Tracim's applications (stand alone and optional), you can add/remove them from `app.enabled` list
```

Reply:
the "(stand alone and optional)" part need to be confirmed by backend dev.
The point is to tell some main features of what is a Tracim's app